dask/dataframe/io/io.py

```diff
@@ -181,15 +191,22 @@ def from_pandas(data, npartitions=None, chunksize=None, sort=True, name=None):
     if not has_parallel_type(data):
         raise TypeError("Input must be a pandas DataFrame or Series")
 
-    if (npartitions is None) == (chunksize is None):
-        raise ValueError("Exactly one of npartitions and chunksize must be specified.")
+    # Infer npartitions if npartions and chunksize is None
+    if (npartitions is None) and (chunksize is None):
+        npartitions = _infer_npartitions(data, "128 MiB")
```

Comment:
Rather than hardcoding 128 MiB, should this be `config['array']['chunk_size']`?

Reply:
Or should we have a different key for dataframe? My initial preference is to reuse the array chunk size, but the 2-D vs. N-D nature of dataframe vs. array may warrant separate keys.

Reply:
As far as I can see there is no key for dataframe. Should I reuse the chunk-size from array?

Reply:
Correct, the choice is between reusing the existing one vs. creating a new key.

I think my preference is to reuse array.chunk_size (make sure to document that).